src/components/PersonLink.tsx

```diff
@@ -0,0 +1,25 @@
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+import { isFemale } from '../helpers';
+
+type Props = {
+  person: Person | string | null | undefined;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  if (typeof person === 'string' || !person) {
+    return <>{person ?? '-'}</>;
+  }
```

Comment:
```js
const NOT_SET_VALUE = '-';
```
```suggestion
  if (!person) {
    return <>{NOT_SET_VALUE}</>;
  }
```

Reply:
You misunderstood the logic behind this.
`person` can be an object of type `Person` or a `string` or `null | undefined`, so I need to check all that cases and treat them differently.